sktime/markov/msm/koopman_reweighted_msm.py

```diff
@@ -2,60 +2,72 @@
 
 import numpy as _np
 from msmtools.estimation import effective_count_matrix
-
-from sktime.markov.msm import MarkovStateModel
+from scipy.sparse import issparse
 from sktime.markov._base import _MSMBaseEstimator
+from sktime.markov.msm import MarkovStateModel
 from sktime.markov.transition_counting import TransitionCountEstimator, TransitionCountModel
 from sktime.util import submatrix
-from ._koopman_reweighted_msm_impl import (bootstrapping_count_matrix, bootstrapping_dtrajs, twostep_count_matrix,
-                                           rank_decision, oom_components, equilibrium_transition_matrix)
 
-__author__ = 'Feliks Nüske, Fabian Paul, marscher'
+from . import _koopman_reweighted_msm_impl as _impl
+from ..util import count_states
+
+__author__ = 'Feliks Nüske, Fabian Paul, marscher, clonker'
 
 
 class KoopmanReweightedMSM(MarkovStateModel):
-    def __init__(self, eigenvalues_OOM=None, sigma=None, omega=None, *args, **kwargs):
-        super(KoopmanReweightedMSM, self).__init__(*args, **kwargs)
-        self._eigenvalues_oom = eigenvalues_OOM
-        self._Xi = oom_components
+    def __init__(self, transition_matrix, c2t=None, oom_components=None,
```

Comment:
does this need docs or am i missing them somewhere?

Reply:
i can guess a lot of them but `ncv` and `c2t` aren't great (I know this wasn't you)

Reply:
oh yeah this needs docs, thanks! agreed on ncv and n2t.